playbooks/roles/openconnect/vars/main.yml

```diff
@@ -26,14 +26,14 @@ ocserv_compilation_directory:  "/usr/local/src/ocserv-{{ ocserv_version }}"
 ocserv_source_filename:        "ocserv-{{ ocserv_version }}.tar.xz"
 ocserv_sig_filename:           "{{ ocserv_source_filename }}.{{ ocserv_sig_extension }}"
 
-mit_gpg_keyserver_address:    "x-hkp://pgp.mit.edu"
+mit_gpg_keyserver_address:     "x-hkp://pgp.mit.edu"
```

Comment:
spurious diff?

Reply:
Just so that it lines up with with the variable above it.